internal/progress.go

```diff
@@ -83,3 +95,31 @@ func (p *Progress) report(firstCall bool) {
 		fmt.Printf("\n")
 	}
 }
+
+func (p *Progress) reportFraction(firstCall bool) {
+	if p.verbose {
+		fmt.Printf("%s: %d/%d\n", p.message, p.progress, p.total)
+		return
+	}
+	if firstCall {
+		fmt.Printf("%s: %d/%d", p.message, p.progress, p.total)
+	} else {
+		delStr := strings.Repeat("\b", countDigits(p.progress-1)+countDigits(p.total)+1)
+		fmt.Printf(delStr+"%d/%d", p.progress, p.total)
+	}
```

Comment:
Assuming that previously reported percentage is `progress-1` since we only call it incrementally during foreign key updates. In the future, we should store previous reported progress and use that to calculate num of digits.